daemon/algod/api/algod.oas2.json

```diff
@@ -1437,17 +1437,17 @@
           "description": "\\[un\\] Name of a unit of this asset, as supplied by the creator. Included only when the name of a unit of this asset is composed of printable utf-8 characters.",
           "type": "string"
         },
-        "unit-name_b64": {
-          "description": "\\[un64\\] base64 encoded name of a unit of this asset, as supplied by the creator.",
```

Comment:
The square brackets document the message pack name, so they aren't needed for these.